### ISSUE TITLE
Difference between CycleGAN and UNIT

### ISSUE THREAD

**Issue by mhusseinsh** [taki0112/UNIT-Tensorflow#5]
Hello,
thanks for this amazing work.

I have a doubt, I want to understand exactly the difference between cycle gan and UNIT in the loss function.
I read your documentation and mostly understand it, but I am still confused in the differences in the loss functions, especially the cycle loss (is it normal than that of the cycle gan) ?
